src/ui/flutter_app/lib/screens/home_drawer.dart

```diff
@@ -300,7 +298,7 @@ class HomeDrawer extends StatelessWidget {
     final stack = Stack(
       children: <Widget>[
         Padding(
-          padding: const EdgeInsets.symmetric(horizontal: 8.0),
+          padding: const EdgeInsets.symmetric(vertical: 8.0),
```

Comment:
This change may have an effect on the style, I need to compile it out and check the effect.